Assets/MixedRealityToolkit/Definitions/Utilities/SolverOrientationType.cs

```diff
@@ -28,6 +28,10 @@ public enum SolverOrientationType
         /// <summary>
         /// Align parallel to the direction the camera is facing 
         /// </summary>
-        CameraAligned
+        CameraAligned,
+        /// <summary>
+        /// Leave the current goal rotation  
+        /// </summary>
```

Comment:
Also not clear how MaintainGoal and Unmodified are fully different. Most of the time GoalRotation will just be transform.rotation. In linked transforms, I guess there would be a difference. But this is extremely confusing to a developer who won’t understand how solvers work.